modules/agent-install-sample-config-bond-sriov.adoc

```diff
@@ -0,0 +1,122 @@
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+The following `agent-config.yaml` file is an example of a manifest for dual port NIC with a bond and SR-IOV interfaces.
+
+[source,yaml]
+----
+apiVersion: v1alpha1
+kind: AgentConfig
+rendezvousIP: 10.10.10.14
+hosts:
+ - hostname: worker-1
+    interfaces:
+     - name: eno1
+       macAddress: 0c:42:a1:55:f3:06
+     - name: eno2
+       macAddress: 0c:42:a1:55:f3:07
+    networkConfig: <1>
+      interfaces: <2>
+        - name: eno1     <3>
+          type: ethernet <4>
+          state: up
+          mac-address: 0c:42:a1:55:f3:06
+          ipv4:
+            enabled: true
+            dhcp: false  <5>
+          ethernet:
+            sr-iov:
+              total-vfs: 2 <6>
+          ipv6:
+            enabled: false
+        - name: sriov:eno1:0
+          type: ethernet
+          state: up  <7>
+          ipv4:
+            enabled: false <8>
+          ipv6:
+            enabled: false
+        - name: sriov:eno1:1
+          type: ethernet
+          state: down
+      - name: eno2
+        type: ethernet
+        state: up
+        mac-address: 0c:42:a1:55:f3:07
+        ipv4:
+          enabled: true
+        ethernet:
+          sr-iov:
+            total-vfs: 2
+        ipv6:
+          enabled: false
+      - name: sriov:eno2:0
+        type: ethernet
+        state: up
+        ipv4:
+          enabled: false
+        ipv6:
+          enabled: false
+      - name: sriov:eno2:1
+        type: ethernet
+        state: down
+      - name: bond0
+        type: bond
+        state: up
+        min-tx-rate: 100 <9>
+        max-tx-rate: 200 <10>
+        link-aggregation:
+          mode: active-backup <11>
+          options:
+            primary: sriov:eno1:0 <12>
+          port:
+            - sriov:eno1:0
+            - sriov:eno2:0
+          ipv4:
+            address:
+            - ip: 10.19.16.57 <13>
+              prefix-length: 23
+            dhcp: false
+            enabled: true
+          ipv6:
+            enabled: false
+          dns-resolver:
+            config:
+              server:
+               - 10.11.5.160
+               - 10.2.70.215
+          routes:
+            config:
+              - destination: 0.0.0.0/0
+                next-hop-address: 10.19.17.254
+                next-hop-interface: bond0 <14>
+                table-id: 254
+----
+<1> The networkConfig field holds the network configuration of the host. It has several subfields, including `interfaces`, `dns-resolver`, and `routes`.
+<2> The `interfaces` field is an array of network interfaces defined for the host.
+<3> Name of the interface.
```

Comment:
I'd recommend trying to be a bit more parallel in these descriptions. Some start with the field name, some are a fragment, some start with an action that the user should take.